src/domain/shared/utils/scroll-utils/ScrollToElement.ts

```diff
@@ -0,0 +1,35 @@
+import { useState } from 'react';
```

Comment:
this file is not a utility function, it's a hook - change the location and file name.